web/src/components/molecules/Common/WorkspaceMenu/index.tsx

```diff
@@ -18,22 +18,12 @@ export interface Props {
   defaultSelectedKeys?: string[];
 }
 
```

Comment:
Any reason you are importing directly from antd for the Menu component and not creating an atom like the other antd components?